scripts/generate-search-config.py

```diff
@@ -3,20 +3,19 @@
 
 At the time of writing, most of the search mapping is written by hand, in
 the search_mapping.json template file. This script adds to that template
-and writes it to a services.json file in the directory provided, which
+and writes it to a <doc_type>.json file in the directory provided, which
 should be reference the 'mappings' directory in a checkout of the Search API.
 
 To preview the mapping that will be generated, do not specify the output path.
 
-Note that the Search API only supports one mapping at a time, which will be
-used for any index relating to any G-Cloud framework. Therefore, care should
-be taken with the release process to ensure that indexing for the currently-
-live framework continues as expected, especially if the new framework's
-mapping is not backward-compatible with the old one.
+Note that most of the digital marketplace code only supports one mapping per
+doc type at a time. Therefore, care should be taken with the release process
```

Comment:
It might be more clear/correct to say "Note that most of the digital marketplace code only supports one index (and therefore one mapping) per doc type at a time."

And then add after this paragraph, perhaps:

>See <https://github.com/alphagov/digitalmarketplace-search-api/blob/master/README.md#updating-the-index-mapping> for more information about how to apply the updated mapping to an index. Backward-incompatible changes to the mapping should be applied by creating a new index, and swapping the index aliases over when ready (for example when deploying a frontend that references the new search manifest).

Reply:
- [ ] do this